### PR TITLE
remove arcgis10x desktop download references

### PR DESCRIPTION
**Pull request by gregbunce**
for more info please see Porter issue 102
https://github.com/agrc/porter/issues/102

### CODE REVIEW

**Comment by gregbunce** [comment]
let me know if I missed anything.

**Comment by gregbunce** [comment]
i'll update the stewardship sheet, too

File: data/address-geocoders-locators/index.html [hunk 1]
```diff
@@ -202,170 +183,3 @@ <h5>Related Resources</h5>
...
 </div>
-<div class="grid package">
-  <div class="grid__col grid__col--12-of-12">
-    <h3 id="RoadsAddressLocatorPackage">Roads Address Locator</h3>
-  </div>
-  <div class="grid__col grid__col--12-of-12 package-content">
-    <p style="display: inline-block">
-      <img
-        style="float: left; margin-right: 1em;"
-        src="{% link images/address-geocode.png %}"
```
Comment (steveoh) [inline data/address-geocoders-locators/index.html:213]: **question**: do we need to check if this image is used anywhere else and can be deleted?

**Review by steveoh — COMMENTED** [review]
looks good. i think we need to check if that image is no longer used and should also be cleaned up.

File: data/address-geocoders-locators/index.html (hunk 1 above)
Comment (gregbunce) [reply data/address-geocoders-locators/index.html:213]: good call.  i did a search and did not find the image being used anywhere else so i removed it from the images folder.

Comment (steveoh) [reply data/address-geocoders-locators/index.html:213]: yep since the build passed you are right. unused.

**Comment by gregbunce** [comment]
@steveoh did you wanna merge this or wait?

**Comment by gregbunce** [comment]
I am happy with it

**Comment by steveoh** [comment]
i'm going to wait until we delete the locators

**Review by stdavis — APPROVED** [review]

**Pull request merged**